os/net/mac/tsch/tsch-slot-operation.c

```diff
@@ -93,9 +93,9 @@
  * of the guard time (one fourth of TSCH_DEFAULT_TS_RX_WAIT) */
 #define SYNC_IE_BOUND ((int32_t)US_TO_RTIMERTICKS(tsch_timing_us[tsch_ts_rx_wait] / 4))
 
-/* By default: check that rtimer runs at >=32kHz and use a guard time of 10us */
-#if RTIMER_SECOND < (32 * 1024)
-#error "TSCH: RTIMER_SECOND < (32 * 1024)"
+/* By default: check that rtimer runs at >=8 kHz and use a guard time of 10us */
```

Comment:
Why did you change this?

Reply:
This comes with my first experiments with cooja polling precise resolu resolution. I wanna use RTIMER_SECOND as resolution, but later i go another way.
Anyway, imho, this change not bad, since code shouldnt stricts it`s conditions more then realy need.
This can be droped with no any suspicions.

Reply:
Yes, please drop this commit - I think we need more justification than that!